### PR TITLE
FIX: Beamline cleanup

### PR DESCRIPTION
**Pull request by slactjohnson**
Fixed some broken devices in MEC's beamline.py, and brought them up to a much more recent pcdsdevices. They ran the last beamtime with this configuration, and had no issues.

### CODE REVIEW

File: mec/beamline.py [hunk 1]
```diff
@@ -32,25 +32,26 @@ def lights_off():
...
-    target_y = epics_motor.Newport('MEC:PPL:MMN:09', name='target y')
-    tgt = TargetXYStage(target_x, target_y, 3.7, 3.5)
+with safe_load('target hexapod'):
+    from .devices import PI_M824_Hexapod 
+    tc_hexapod = PI_M824_Hexapod('MEC:HEX:01', name='target hexapod') 
+
+    from .devices import TargetStage
+    target = TargetStage(target_x, tc_hexapod, 0.75, 0.75)
+
+#with safe_load('target xy stage'):
```
Comment (sfsyunus) [inline mec/beamline.py:42]: Is this the relic stage stack from when the target hexapod was out of commission for repairs?

File: mec/beamline.py [hunk 2]
```diff
@@ -69,15 +70,16 @@ def lights_off():
...
-    #mec_yag0 = mec_pim1
-    mec_yag0 = PIM('HXX:UM6:MMS:08', prefix_det='HXX:UM6:CVV:01', name='mec yag0')# ==works
-    mec_yag1 = PIM('MEC:HXM:MMS:16', prefix_det='MEC:HXM:CVV:01', name='mec yag1')# ==works
-    mec_yag2 = PIMMotor('MEC:XT2:MMS:13', name='mec yag2')
-    #mec_yag2 = PIM('MEC:XT2:MMS:13', prefix_det='MEC:XT2:CVV:01', name='mec yag2')# !=works
-    mec_yag3 = PIMMotor('MEC:XT2:MMS:29', name='mec yag3')
-    #mec_yag3 = PIM('MEC:XT2:MMS:29', prefix_det='MEC:XT2:CVV:02', name='mec yag3')# !=works
+    from pcdsdevices.pim import PIM, PIMY
+    mec_yag0 = PIMY('HXX:HXM:PIM', name='mec yag0')
+    yag0 = mec_yag0
```
Comment (sfsyunus) [inline mec/beamline.py:75]: This is neat! Does it go looking in the camviewer.cfg to find the PV?

**Review by sfsyunus — APPROVED** [review]
Looks great! Thank you

File: mec/beamline.py (hunk 1 above)
Comment (slactjohnson) [reply mec/beamline.py:42]: That looks like the target stack from when the hexapod was functional.

File: mec/beamline.py (hunk 2 above)
Comment (slactjohnson) [reply mec/beamline.py:75]: No, that would be cool though. The PIMY class omits the camera, and just gives you control of the stage. The camera IOC on this PIM is not compatible with the normal PIM class. I don't think MEC uses the PIM camera functionality in python anyway, so I think this is OK for now until MEC's YAGs get a bit more standardized.

**Pull request merged**